Graphs/Table/index.js

```diff
@@ -667,7 +702,10 @@ const TableGraph = (props) => {
                                         return (
                                             <Table
                                                 ref={registerChild}
-                                                onRowsRendered={onRowsRendered}
+                                                onRowsRendered={({ startIndex }) => {
+                                                    currentStartIndex = startIndex;
+                                                    return onRowsRendered
```

Comment:
this prop is a callback function.
to maintain parity with previous version, this should not return anything, instead should invoke InfiniteLoader's onRowsRendered()
```
onRowsRendered={(props) => {
    setCurrentStartIndex(props.startIndex);
    onRowsRendered(props);
}}
```